### DISCUSSION TITLE
What is the small green line by the tab title?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7666] [Q&A] [answered]
I took a picture so you know what I mean: [imgbb.co](https://ibb.co/MhFTrZ5)
I'd like to hide this (even if I can just change the color to match the background if I can) but can't figure out how in the settings
I've also looked through the source code a little bit looking for anything that jumped out but couldn't find anything.
Surely this isn't what tabs.indicator is, right?
Any info would be appreciated, thanks :)

**Comment by toofar** [comment, marked as answer]
> Surely this isn't what tabs.indicator is, right?

It is! I guess it indicates the page load progress. You can hide it by setting `tabs.indicator.width` to 0.
If you want to change the color the `colors.tabs.indicator.start/stop` settings describe two colour points to pick colours from according to the page load percentage.
